rele/subscription.py

```diff
@@ -32,7 +33,8 @@ def __init__(self, subscription, suffix=None):
     def __call__(self, message):
         db.close_old_connections()
 
-        logger.info(f'Start processing message for {self._subscription}')
+        logger.info(f'Start processing message for {self._subscription}',
+                    extra=self._build_task_info())
```

Comment:
There is no concept of "task" here. I'd call it `_build_metrics()` or `_build_sub_info()`

Reply:
Done